src/main/webapp/profile.js

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Fetch info about the spceific user logged in 
+function displayProfile() {
+    // Get the userID parameter from the URL
+    var params = new URL(location.href).searchParams;
+    var userID = params.get('userID');
+    console.log(userID);
+
+    fetch('/profile?user-id='+userID).then(response => response.json()).then((user) => {
```

Comment:
What happens if there is no user logged in? Is the My Account link still visible from other page?

Reply:
In the displayLoginLogoutLink function in registration.js I have it so the button will only display when the user is logged in. I just included this in that function since I thought it would be easier to have all login/logout display logic together.